dwave/system/composites/tiling.py

```diff
@@ -39,25 +39,40 @@
 
```

Comment:
Lines 16-23 also need updating. Perhaps something like:

A :std:doc:`dimod composite <oceandocs:docs_dimod/reference/samplers>` that tiles small problems
multiple times to a structured sampler.
The :class:`.TilingComposite` class takes a problem that can fit on a small
:term:`Chimera` graph and replicates it across a larger Pegasus or
Chimera graph to obtain samples from multiple areas of the solver in one call.
For example, a 2x2 Chimera lattice could be tiled 64 times (8x8) on a fully-yielded
D-Wave 2000Q system (16x16).